App/VSTS/FollowButton.tsx

```diff
@@ -0,0 +1,66 @@
+/// <reference path="../../office.d.ts" />
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import { changeFollow } from '../actionsEZ';
+import { connect } from 'react-redux';
+import { FollowStateTypes } from '../statesEZ';
+
+let buttonText: string = '';
+let isDisabled: boolean = false;
+let iconType: string = 'ms-Icon ms-Icon--eye';
+
+interface IFollowButtonProps {
+  dispatch?: any;
+  followState?: FollowStateTypes;
+}
+
+function mapStateToProps(state: any): IFollowButtonProps {
+  return {
+    followState: state.quickActionState.followState,
```

Comment:
I changed the name to workItemState - this will be addressed again after the merge of components